splunk_add_on_ucc_framework/generators/conf_files/create_inputs_conf.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, Dict, List
+
+from splunk_add_on_ucc_framework.generators.conf_files import ConfGenerator
+from splunk_add_on_ucc_framework.global_config import GlobalConfig
+
+
+class InputsConf(ConfGenerator):
+    __description__ = (
+        "Generates `inputs.conf` and `inputs.conf.spec` "
+        "file for the services mentioned in globalConfig"
+    )
+
+    def __init__(
+        self,
+        global_config: GlobalConfig,
+        input_dir: str,
+        output_dir: str,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(global_config, input_dir, output_dir, **kwargs)
+        self.conf_file = "inputs.conf"
+
+    def _set_attributes(self, **kwargs: Any) -> None:
```

Comment:
Yes, we could merge `_set_attributes` and `__init__`, I kept them different as to define what needs to be initialized and what attributes need to be set for `self` for each individual classes. Should we merge them in `__init__`?

Currently, `**kwargs` is required by AppConf class to generate the details required for `app.conf` which comes from `build.py` (to avoid redoing of things), and perhaps in future for XML or Python or other file generation, we might require it. What are your thoughts on this?

Reply:
We could for example at least leave `def _set_attributes(self, **kwargs):` but change the method signature in subclasses to for example: `def _set_attributes(self):` (when no parameters are needed), or `def _set_attributes(self, some_param):` - type checking should accept that, I think and we'll get better error messages when some parameter is missing.

Reply:
Since the parent class is invoking the `_set_attributes`, it will pass `**kwargs` to whichever child class requires it. Moreover, I tried updating the method declaration to `def _set_attributes(self)` where `**kwargs` aren't required, and it raised a TypeError.